logservice.go

```diff
@@ -69,12 +70,16 @@ func parseFlags() app {
 		os.Exit(0)
 	}
 
-	if len(a.url) == 0 {
-		a.url = os.Getenv("SD_API_URI")
+	if len(a.storeUrl) == 0 {
```

Comment:
This looks like it can be combined with line 81

Reply:
Can check if `SD_API_URL` is set. If yes, then assign. Otherwise, log error and `flag.Usage()`.